SpatialGDK/Source/SpatialGDK/Public/Utils/SpatialDebugger.h

```diff
@@ -48,6 +49,17 @@ struct FWorkerRegionInfo
 	FBox2D Extents;
 };
 
+UENUM()
+namespace EActorTagDrawMode
+{
+enum Type
+{
+	None,
+	LocalPlayer,
+	All
+};
+}
```

Comment:
I think running clang-format should do this but
```suggestion
} // namespace EActorTagDrawMode
```

Reply:
Doesn't seem to. Gotta do everything yourself around here!